openedx/core/djangoapps/credit/email_utils.py

```diff
@@ -42,19 +44,31 @@ def send_credit_notifications(username, course_key):
 
     course = modulestore().get_course(course_key, depth=0)
     course_display_name = course.display_name
-    branded_logo = dict(title='Logo', path=settings.NOTIFICATION_EMAIL_EDX_LOGO, cid=str(uuid.uuid4()))
     tracking_context = tracker.get_tracker().resolve_context()
     tracking_id = str(tracking_context.get('user_id'))
     client_id = str(tracking_context.get('client_id'))
     events = '&t=event&ec=email&ea=open'
     tracking_pixel = 'https://www.google-analytics.com/collect?v=1&tid' + tracking_id + '&cid' + client_id + events
     dashboard_link = _email_url_parser('dashboard')
-    credit_course_link = _email_url_parser('courses', "?type=credit")
+    credit_course_link = _email_url_parser('courses', '?type=credit')
+
+    # get attached branded logo
+    logo_image = cache.get('attached-logo')
```

Comment:
Could we namespace this cache key to avoid an accidental collision with another part of the system?  Usually, we use the name of the app, so something like "credit.email.attached-logo"?